wiki/Help_Centre/Restricted_Mode/en.md

```diff
@@ -0,0 +1,58 @@
+- **How you obtained the means to do so.** This applies mostly to cheating - we want to know where you obtained the tool you used, how you came across it, and whether or not it is still present on your computer.
+- **Why we should allow you back.** Most of the time, this ties in to the second point. We just want to ensure that you will not be breaking the rules again! Please understand that the heavier your account's history, the more we will expect from this section.
+
+## My appeal has been denied!
+
+There are only a few reasons why an appeal gets denied.
+- **It was sent too early.** If the cooldown for your account has not yet passed, your appeal will be denied. If you are unsure when you can actually appeal, please reply to our answer - we will give you the correct date.
+- **Your appeal was insufficient or incomplete.** If your appeal does not cover the four points listed above, we will request that you write a more proper one. This **does not** reset the cooldown for your account and you are free to send your new appeal at any point in time! If you attempted lying or hiding parts of your history (mainly multi-accounting or cheating), your appeal will be denied for being non-truthful and your cooldown reset to three months. This is another chance for you to reflect on your actions and make sure you did not forget anything.
+- **You have been evading or playing on other accounts.** Creating another account to be able to play online while your account is restricted is an attempt at evading the restriction and will reset your cooldown to three months from the date you created the account. If you attempt evading too many times, we might permanently deny the appeal. If you are still accessing your multi-accounts while restricted, your cooldown will also be reset. Remember that multi-accounting is against the [rules](/wiki/rules) and you are expected to only play on your original account. If you are unsure which account you should be playing on, send us an email and we will tell you!
+- **You continued offending while restricted.** Why would we unrestrict someone who is still breaking the rules after having been restricted? If you do continue offending, the cooldown for your account will be reset to your last offence. If this happens too much, your appeal will be permanently denied.
+- **Your history is too severe.** In cases of extreme offenders, or people who get restricted again after already being back on appeal, we might simply state that your history is too severe to rejoin the game at the current time. In those cases, we recommend to wait **a long time** before appealing again, and making sure you do not continue breaking the rules.
```

Comment:
`- **Your history is too severe.** In cases of extreme offenders, or players who get restricted again after they have already been unrestricted on a successful appeal, we might simply state that your history is too severe to rejoin the game at the current time. Should this apply to you, we recommend waiting **a long time** before appealing again, and making sure you do not break any further rules in that time.`